_tools/cite-guide.md

```diff
@@ -0,0 +1,459 @@
+---
+layout: default
+title: CITE Participation Guide
+collection: tools
+permalink: /tools/citeguide/
+---
+
+# Community Interoperability Test Environment (CITE) Participation Guidance
+
+Prepared By: The FPKI Technical Working Group (TWG)  <br>
+An FPKI Policy Authority Working Group
+
+Last Update: February 12th, 2019 <br>
+
+## Overview
+

```

Comment:
Need to have links to each section below for easy navigation.

Reply:
Added.